recipes/eumdac/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "eumdac" %}
+{% set version = "2.0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: bc320d6ca7ce9f192004de01663ffa91c98ac587638541eee80b5608b733efb8
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install ."
+
+requirements:
+  host:
+    - pip
+    - python
```

```suggestion
    - python >=3.7
```
Upstream's setup.py suggests 3.7 as minimal Python version and 3.9 as maximal. Is that correct?